Resources/doc/reference/command.rst

```diff
@@ -0,0 +1,28 @@
+Command
+=======
+
+You can create a dump file with products schema information. Even those created with the previous command:
+
+.. code-block:: bash
+
+    Usage:
+     sonata:doctrine:utils -r "/ProductBundle/" -f /tmp/dump.json
+
+    Options:
+     --filename (-f)       If filename is specified, result will be dump into this file under json format.
+     --entity-name (-E)    If entity-name is set, dump will only contain the specified entity and all it's extended classes.
+     --regex (-r)          If regex is set, dump will only contain entities which name match the pattern.
+     --help (-h)           Display this help message.
+     --quiet (-q)          Do not output any message.
+     --verbose (-v|vv|vvv) Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug.
+     --version (-V)        Display this application version.
+     --ansi                Force ANSI output.
+     --no-ansi             Disable ANSI output.
+     --no-interaction (-n) Do not ask any interactive question.
+     --shell (-s)          Launch the shell.
+     --process-isolation   Launch commands from shell as a separate process.
+     --env (-e)            The Environment name. (default: "dev")
+     --no-debug            Switches off debug mode.
+
+
+This can be used for import/export purposes for example.
```

Comment:
``for example`` is useless. Also, you need to be more verbose about what this command does.